framework/db/QueryTrait.php

```diff
@@ -341,7 +341,7 @@ protected function normalizeOrderBy($columns)
         if (is_array($columns)) {
             return $columns;
         } else {
-            $columns = preg_split('/\s*,\s*/', trim($columns), -1, PREG_SPLIT_NO_EMPTY);
+            $columns = preg_split('/\s*,(?![^\(]*\))\s*/', trim($columns), -1, PREG_SPLIT_NO_EMPTY);
```

Comment:
Actually, it's not grouping. it's `if-else` syntax for regular expression
http://www.regular-expressions.info/conditional.html